internal/v2/models/resource_integration.go

```diff
@@ -150,13 +153,14 @@ func getConnectedResourceTypes(ctx context.Context, model ResourceIntegrationMod
 
 func getOwnerConfig(ctx context.Context, ownerConfig *OwnerConfig) (client.UpsertOwnerV4, error) {
 	var values []string
-	diags := ownerConfig.Values.ElementsAs(ctx, &values, false)
+
```

Comment:
so this is how it's going to work if a user uses the old format:
1. apply or plan will fail - because we added new required fields.
2. user adds the new fields (same values) and remove the old fields. (or in other words, renames the fields).
3. run apply again -- 'y' and done.